src/database/queries/noob.js

```diff
@@ -0,0 +1,61 @@
+import knex from '../knex'
+
+import {
+  createRecord,
+  findRecord,
+  updateRecord,
+  deleteRecord,
+  findAllWhere,
+  findAll } from './utilities'
+
+const createNoob = attributes =>
```

Comment:
can you rename all these functions to not have `noob` in them? They should just be called `findByHandle`, `updateByHandle` etc. Since this is the `noob.js`  file I know all these functions are for noobs.